csvcubed/csvcubed/cli/entrypoint.py

```diff
@@ -0,0 +1,83 @@
+"""
+CLI
+---
+The *Command Line Interface* for :mod:`~csvcubed.csvcubedcli.infojson2csvqb`.
+"""
+from email.policy import default
+import logging
+import click
+from pathlib import Path
+import colorama
+from csvcubed.utils.log import start_logging
+
+from .build import build
+
+
+@click.group(context_settings=dict(help_option_names=["-h", "--help"]))
+def entry_point():
+    """
+    csvcubed - a tool to generate qb-flavoured CSV-W cubes from COGS-style info.json files.
+    """
+    colorama.init(autoreset=True, wrap=True)
+
+
+@entry_point.command("build")
+@click.option(
+    "--config",
+    "-c",
+    help="Location of the info.json file containing the QB column mapping definitions.",
+    type=click.Path(exists=True, path_type=Path, file_okay=True, dir_okay=False),
+    required=True,
+    metavar="CONFIG_PATH",
+)
+@click.option(
+    "--out",
+    "-o",
+    help="Location of the CSV-W outputs.",
+    default="./out",
+    show_default=True,
+    type=click.Path(path_type=Path, file_okay=False, dir_okay=True),
+    metavar="OUT_DIR",
+)
+@click.option(
+    "--fail-when-validation-error/--ignore-validation-errors",
+    help="Fail when validation errors occur or ignore validation errors and continue generating a CSV-W.",
+    default=True,
+    show_default=True,
+)
+@click.option(
+    "--validation-errors-to-file",
+    "validation_errors_to_file",
+    help="Save validation errors to a `validation-errors.json` file in the output directory.",
+    flag_value=True,
+    default=False,
+    show_default=True,
+)
+@click.option("--logginglvl",
+    help= "select a logging level out of: 'warn', 'err' or 'crit'.",
+    type=click.Choice(['warn', 'err', 'crit'], case_sensitive=False),
```

Comment:
This seems like a nice way to select different levels. Can we add `debug` and `info` to the list?